app/actors/Actors.scala

```diff
@@ -1,47 +1,46 @@
 package actors
 
+import javax.inject._
+
+import akka.actor.{ActorRef, ActorSystem}
+import com.google.inject.AbstractModule
 import play.api._
-import play.api.libs.concurrent.Akka
+import play.api.libs.concurrent.AkkaGuiceSupport
 import backend._
 import akka.cluster.Cluster
 import java.net.URL
 
 /**
- * Lookup for actors used by the web front end.
+ * Guice module that provides actors.
+ *
+ * Registered in application.conf.
  */
-object Actors {
-
-  private def actors(implicit app: Application) = app.plugin[Actors]
-    .getOrElse(sys.error("Actors plugin not registered"))
-
-  /**
-   * Get the region manager client.
-   */
-  def regionManagerClient(implicit app: Application) = actors.regionManagerClient
+class Actors extends AbstractModule with AkkaGuiceSupport {
+
+  def configure() = {
+    // Bind the region manager client
+    bindActor[RegionManagerClient]("regionManagerClient")
```

Comment:
Looks pretty slick indeed (and I usually avoid those like fire)! 
Does the `@Named` also work with deep paths or only for top level actors? The first is probably even doable, but I just thought that it may encourage weird tangling.